docs/source/conf.py

```diff
@@ -88,6 +88,7 @@
 #
 # html_theme_options = {}
 html_theme_options = {
+    'analytics_id': 'UA-17296547-4',
```

Comment:
unrelated change

Reply:
Indeed - I didn't see the worth in creating a PR for this but if insist I can do it.

Reply:
ok